.evergreen/config.yml

```diff
@@ -1377,39 +1377,47 @@ tasks:
     commands:
       - func: run-make
         vars:
-          targets: "build"
+          targets: "build build-tests"
           BUILD_ENV: "PATH=/opt/golang/go1.9/bin:$PATH GOROOT=/opt/golang/go1.9"
 
   - name: go1.10-build
     tags: ["compile-check"]
     commands:
       - func: run-make
         vars:
-          targets: "build"
+          targets: "build build-tests"
           BUILD_ENV: "PATH=/opt/golang/go1.10/bin:$PATH GOROOT=/opt/golang/go1.10"
 
+  # Build with whatever the latest Go version is that we're using for tests
+  - name: build
+    tags: ["compile-check"]
+    commands:
+      - func: run-make
+        vars:
+          targets: "build build-tests"
+
   - name: linux-32-bit
     tags: ["compile-check"]
     commands:
       - func: run-make
         vars:
-          targets: "build"
+          targets: "build-no-tags"
```

Comment:
The `gssapi` tag caused build issues on these cross-compiled builds because it wasn't finding any Go files to compile in the `auth/internal/gssapi` package. AFAIK, these tasks were meant to be quick checks to make sure we weren't breaking anything for cross-compilations and those files are already compiled in other tasks, so the simplest solution was to do these compilations without tags.